### PR TITLE
Fix workflow for ec2 linux integration test

### PR DESCRIPTION
**Pull request by ChenaLee**
Changes for the git workflow files in the main repo here: https://github.com/ChenaLee/amazon-cloudwatch-agent/tree/e2e_test_package_separation.
I did thinks those files need to live there because many steps are using SHA of the agent repo commit.

### CODE REVIEW

File: README.md
Comment (SaxyPandaBear) [inline README.md:2]: Don't think this was intentional

File: terraform/ec2/localstack/main.tf
Comment (SaxyPandaBear) [inline terraform/ec2/localstack/main.tf:50]: So how this all worked in the main repo is that part of the workflow builds the binaries and executables and uploads them to S3. This is a prerequisite step to executing the integration tests. How are we planning on keeping integration test executions up to date here when the main agent code lives in the other repo? This terraform is very likely to change since we're ripping it out from where the rest of the agent code is.

File: README.md
Comment (ChenaLee) [reply README.md:2]: no, thanks

File: terraform/ec2/localstack/main.tf
Comment (ChenaLee) [reply terraform/ec2/localstack/main.tf:50]: Not following. main repo github actions steps will check out this repo and use these terraform files. Every time either of them changes if changes are incompatible main repo's github workflow should fail. Am I missing something?

Comment (SaxyPandaBear) [reply terraform/ec2/localstack/main.tf:50]: Okay then I misunderstood how this was going to be used. Makes sense. Disregard.

**Review by SaxyPandaBear — APPROVED** [review]

**Comment by SaxyPandaBear** [comment]
Are you manually requesting review from amazon-auto? I think that's incorrect. If we want, we can request review from our GitHub team. We do this in the main repo via CODEOWNERS

**Comment by ChenaLee** [comment]
> Are you manually requesting review from amazon-auto? I think that's incorrect. If we want, we can request review from our GitHub team. We do this in the main repo via CODEOWNERS

Yes, I didn't really know what that was though. How can I make sure people get notifications for review requests? Separate page for that?

**Pull request merged**
